src/cards/repo-card.js

```diff
@@ -49,9 +49,11 @@ const getBadgeSVG = (label, textColor) => `
  *
  * @param {RepositoryData} repo Repository data.
  * @param {Partial<RepoCardOptions>} options Card options.
+ * @param {number} card_width The width of the card.
+ * @param {number} card_height The height of the card.
  * @returns {string} Repository card SVG object.
  */
-const renderRepoCard = (repo, options = {}) => {
+const renderRepoCard = (repo, options = {card_width,card_height}) => {
```

Comment:
```suggestion
const renderRepoCard = (repo, options = {}) => {
```

This is unnecessary since you use object restructuring on lines 78-79.